paparazzi/src/test/java/app/cash/paparazzi/internal/resources/PseudolocaleGeneratorTest.kt

```diff
@@ -81,10 +81,15 @@ class PseudolocaleGeneratorTest {
     // STRING
     with(table[ResourceNamespace.TODO(), ResourceType.STRING]!!) {
       assertThat(size).isEqualTo(4)
-      assertThat(getValue("string_name").value).isEqualTo("${bidiWordStart}Test$bidiWordEnd ${bidiWordStart}String$bidiWordEnd")
-      assertThat(getValue("string_name_xliff").value).isEqualTo("${bidiWordStart}Test$bidiWordEnd ${bidiWordStart}String$bidiWordEnd $bidiWordStart{0}$bidiWordEnd ${bidiWordStart}with$bidiWordEnd ${bidiWordStart}suffix$bidiWordEnd")
-      assertThat(getValue("string_name_html").value).isEqualTo("${bidiWordStart}This$bidiWordEnd ${bidiWordStart}Test$bidiWordEnd ${bidiWordStart}String$bidiWordEnd")
-      assertThat(getValue("string_name_html").rawXmlValue).isEqualTo("${bidiWordStart}This$bidiWordEnd $bidiWordStart<b>Test</b>$bidiWordEnd ${bidiWordStart}String$bidiWordEnd")
+      assertThat(getValue("string_name").value)
+        .isEqualTo("${bidiWordStart}Test$bidiWordEnd ${bidiWordStart}String$bidiWordEnd")
+      @Suppress("ktlint:standard:max-line-length")
+      assertThat(getValue("string_name_xliff").value)
+        .isEqualTo("${bidiWordStart}Test$bidiWordEnd ${bidiWordStart}String$bidiWordEnd $bidiWordStart{0}$bidiWordEnd ${bidiWordStart}with$bidiWordEnd ${bidiWordStart}suffix$bidiWordEnd")
+      assertThat(getValue("string_name_html").value)
+        .isEqualTo("${bidiWordStart}This$bidiWordEnd ${bidiWordStart}Test$bidiWordEnd ${bidiWordStart}String$bidiWordEnd")
+      assertThat(getValue("string_name_html").rawXmlValue)
+        .isEqualTo("${bidiWordStart}This$bidiWordEnd $bidiWordStart<b>Test</b>$bidiWordEnd ${bidiWordStart}String$bidiWordEnd")
```

Comment:
what changes led to these?  revert?

Reply:
Most of these "led to these" questions were early failures because the intellij style DOES NOT 😲 define any max line length. I since then learned this fact and that the "now-default" official ktlint style defines 140. I think it's not a question to stick to intellij/android studio style, but it's a question whether there should be some maximum line length, even if it's >140?

Reply:
There shouldn't be any diff right? That's why we have the Spotless Gradle plugin and not an IDE-enforcement.

You can have a go, but I don't recommend it, I wasted 4-5 hours on Sunday, and what knowledge I gained might help me to redo these quick when I have time (this week is very full).

In case you go ahead: the biggest mistake I made was not setting the `intellij_idea` style right off the bat, that lead to some of these changes, which I included because they made sense to improve things a bit. Feel free to have a go after these warnings :) and cherry-pick anything you find useful here.

Reply:
btw, I used latest 1.0.1